yarn.lock

```diff
@@ -0,0 +1,1317 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
I think we should gitignore this `yarn.lock` file. I don't think you usually include it for libraries/packages because it might interfere with yarn resolution for whichever application is using the package.

Reply:
Looks like the `yarn.lock` file is still in the diff. Might need to squash everything down to one commit and make sure the `yarn.lock` file is not included in that diff.